sdk/src/beta9/abstractions/base/runner.py

```diff
@@ -267,6 +267,7 @@ def prepare_runtime(
                 self.image_available = True
                 self.image_id = image_build_result.image_id
             else:
+                terminal.error("Image build failed", exit=False)
```

Comment:
Don't we want to exit in these failure cases?

Reply:
I don't think so. Not for all cases. Take the taskqueue wrapper for example.

```
    def put(self, *args, **kwargs) -> bool:
        if not self.parent.prepare_runtime(
            func=self.func,
            stub_type=TASKQUEUE_STUB_TYPE,
        ):
            return False
```

Reply:
I'm just adding messaging to these validations. I think it'll be fine, but it might show up in the logs if there are errors for things like taskqueue.put.